# A Câmara de Vereadores e o COVID-19

Você também fica curioso(a) para saber o que a Câmara de Vereadores
de Feira de Santana fez em relação ao COVID-19? O que discutiram?
O quão levaram a sério o vírus? Vamos responder essas perguntas e
também te mostrar como fazer essa análise. Vem com a gente!

Desde o início do ano o mundo tem falado a respeito do COVID-19.
Por isso, vamos basear nossa análise no período de 1 de Janeiro
a 27 de Março de 2020. Para entender o que se passou na Câmara
vamos utilizar duas fontes de dados:

* [Diário Oficial](diariooficial.feiradesantana.ba.gov.br/)
* [Atas das sessões](https://www.feiradesantana.ba.leg.br/atas)

Nas atas temos acesso ao que foi falado nos discursos e no Diário
Oficial temos acesso ao que realmente virou (ou não) lei. Você
pode fazer _download_ dos dados [aqui]()
e [aqui](https://www.kaggle.com/anapaulagomes/dirios-oficiais-de-feira-de-santana).
Procuramos mas não foi encontrada nenhuma menção ao vírus na
[Agenda da Câmara Municipal](https://www.feiradesantana.ba.leg.br/agenda).

Lembrando que as atas foram disponibilizadas no site da casa apenas
depois de uma reunião nossa com o presidente em exercício José Carneiro.
Uma grande vitória dos dados abertos e da transparência na cidade.

Os dados são coletados por nós e todo código está [aberto e disponível
no Github](https://github.com/DadosAbertosDeFeira/maria-quiteria/).

## Vamos começar por as atas

As atas trazem uma descrição do que foi falado durante as sessões.
Se você quer acompanhar o posicionamento do seu vereador(a), é uma
boa maneira. Vamos ver se encontramos alguém falando sobre *coronavírus*
ou *vírus*.

Se você não é uma pessoa técnica, não se preocupe. Vamos continuar o texto
junto com o código.

In [47]:
import pandas as pd


# esse arquivo pode ser baixado aqui: https://www.kaggle.com/anapaulagomes/atas-da-cmara-de-vereadores
atas = pd.read_csv('atas-28.03.2020.csv')
atas.describe()

crawled_at  \
count                          238   
unique                         238   
top     2020-03-28 07:57:52.042252   
freq                             1   

                                                                crawled_from  \
count                                                                    238   
unique                                                                    18   
top     https://www.feiradesantana.ba.leg.br/atas?mes=10&ano=2019&Acessar=OK   
freq                                                                      26   

              date        event_type  \
count          238               238   
unique         177                 4   
top     2020-03-10  sessao_ordinaria   
freq             2               140   

                                                                                               file_content  \
count                                                                                                   238   
unique                                                                                                  237   
top     \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
freq                                                                                                      2   

                                                                file_urls  \
count                                                                 238   
unique                                                                238   
top     https://www.feiradesantana.ba.leg.br/admin/atas/5e4d43cf4d73d.pdf   
freq                                                                    1   

                             title  
count                          238  
unique                         216  
top     Ata da 8ª Sessão Ordinária  
freq                             3

Explicando um pouco sobre os dados (colunas):

* `crawled_at`: data da coleta
* `crawled_from`: fonte da coleta (site de onde a informação foi retirada)
* `date`: data da sessão
* `event_type`: tipo do evento: sessão ordinária, ordem do dia, solene, etc
* `file_content`: conteúdo do arquivo da ata
* `file_urls`: url(s) do arquivo da ata
* `title`: título da ata

### Filtrando por data

Bom, vamos então filtrar os dados e pegar apenas as atas entre 1 de Janeiro
e 28 de Março:

In [48]:
atas["date"] = pd.to_datetime(atas["date"])
atas["date"]
atas = atas[atas["date"].isin(pd.date_range("2020-01-01", "2020-03-28"))]
atas = atas.sort_values('date', ascending=True)  # aqui ordenados por data em ordem crescente
atas.head()

crawled_at  \
89   2020-03-28 07:57:39.693437   
100  2020-03-28 07:57:41.698716   
112  2020-03-28 07:57:43.192076   
124  2020-03-28 07:57:44.938683   
135  2020-03-28 07:57:46.565511   

                                                            crawled_from  \
89   https://www.feiradesantana.ba.leg.br/atas?mes=2&ano=2020&Acessar=OK   
100  https://www.feiradesantana.ba.leg.br/atas?mes=2&ano=2020&Acessar=OK   
112  https://www.feiradesantana.ba.leg.br/atas?mes=2&ano=2020&Acessar=OK   
124  https://www.feiradesantana.ba.leg.br/atas?mes=2&ano=2020&Acessar=OK   
135  https://www.feiradesantana.ba.leg.br/atas?mes=2&ano=2020&Acessar=OK   

          date        event_type  \
89  2020-02-03  sessao_ordinaria   
100 2020-02-04  sessao_ordinaria   
112 2020-02-10  sessao_ordinaria   
124 2020-02-11  sessao_ordinaria   
135 2020-02-12  sessao_ordinaria   

                                                                                            file_content  \
89   \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
100  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
112  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
124  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
135  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   

                                                             file_urls  \
89   https://www.feiradesantana.ba.leg.br/admin/atas/5e46a0f937369.pdf   
100  https://www.feiradesantana.ba.leg.br/admin/atas/5e46a0c72ab2c.pdf   
112  https://www.feiradesantana.ba.leg.br/admin/atas/5e4400067cbf8.pdf   
124  https://www.feiradesantana.ba.leg.br/admin/atas/5e4400740c359.pdf   
135  https://www.feiradesantana.ba.leg.br/admin/atas/5e4a7ec5b8961.pdf   

                          title  
89   Ata da 1ª Sessão Ordinária  
100  Ata da 2ª Sessão Ordinária  
112  Ata da 3ª Sessão Ordinária  
124  Ata da 4ª Sessão Ordinária  
135  Ata da 5ª Sessão Ordinária

Bom, quantas atas temos entre Janeiro e Março?

In [49]:
len(atas)

21

Apenas 21 atas, afinal os trabalhos na casa começaram no início de Fevereiro
e pausaram por uma semana por conta do coronavírus.

Fonte:
https://www.feiradesantana.ba.leg.br/agenda
https://www.feiradesantana.ba.leg.br/noticia/2029/c-mara-municipal-suspende-sess-es-ordin-rias-da-pr-xima-semana

### Filtrando conteúdo relacionado ao COVID-19

Agora que temos nossos dados filtrados por data, vamos ao conteúdo.
Na coluna `file_content` podemos ver o conteúdo das atas. Nelas vamos
buscar por as palavras:

- COVID, COVID-19
- coronavírus, corona vírus

In [50]:
termos_covid = "COVID-19|coronavírus"
atas = atas[atas['file_content'].str.contains(termos_covid, case=False)]
atas

crawled_at  \
100  2020-03-28 07:57:41.698716   
135  2020-03-28 07:57:46.565511   
145  2020-03-28 07:57:48.739193   
155  2020-03-28 07:57:50.723329   
165  2020-03-28 07:57:52.208402   
101  2020-03-28 07:57:41.765690   
113  2020-03-28 07:57:43.292739   
136  2020-03-28 07:57:46.803851   
146  2020-03-28 07:57:49.094699   
156  2020-03-28 07:57:50.945742   
166  2020-03-28 07:57:52.328537   
176  2020-03-28 07:57:53.417289   

                                                            crawled_from  \
100  https://www.feiradesantana.ba.leg.br/atas?mes=2&ano=2020&Acessar=OK   
135  https://www.feiradesantana.ba.leg.br/atas?mes=2&ano=2020&Acessar=OK   
145  https://www.feiradesantana.ba.leg.br/atas?mes=2&ano=2020&Acessar=OK   
155  https://www.feiradesantana.ba.leg.br/atas?mes=2&ano=2020&Acessar=OK   
165  https://www.feiradesantana.ba.leg.br/atas?mes=2&ano=2020&Acessar=OK   
101  https://www.feiradesantana.ba.leg.br/atas?mes=3&ano=2020&Acessar=OK   
113  https://www.feiradesantana.ba.leg.br/atas?mes=3&ano=2020&Acessar=OK   
136  https://www.feiradesantana.ba.leg.br/atas?mes=3&ano=2020&Acessar=OK   
146  https://www.feiradesantana.ba.leg.br/atas?mes=3&ano=2020&Acessar=OK   
156  https://www.feiradesantana.ba.leg.br/atas?mes=3&ano=2020&Acessar=OK   
166  https://www.feiradesantana.ba.leg.br/atas?mes=3&ano=2020&Acessar=OK   
176  https://www.feiradesantana.ba.leg.br/atas?mes=3&ano=2020&Acessar=OK   

          date        event_type  \
100 2020-02-04  sessao_ordinaria   
135 2020-02-12  sessao_ordinaria   
145 2020-02-17  sessao_ordinaria   
155 2020-02-18  sessao_ordinaria   
165 2020-02-19  sessao_ordinaria   
101 2020-03-02  sessao_ordinaria   
113 2020-03-03  sessao_ordinaria   
136 2020-03-09  sessao_ordinaria   
146 2020-03-10  sessao_ordinaria   
156 2020-03-11  sessao_ordinaria   
166 2020-03-16  sessao_ordinaria   
176 2020-03-17  sessao_ordinaria   

                                                                                            file_content  \
100  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
135  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
145  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
155  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
165  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
101  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
113  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
136  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
146  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
156  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
166  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
176  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   

                                                             file_urls  \
100  https://www.feiradesantana.ba.leg.br/admin/atas/5e46a0c72ab2c.pdf   
135  https://www.feiradesantana.ba.leg.br/admin/atas/5e4a7ec5b8961.pdf   
145  https://www.feiradesantana.ba.leg.br/admin/atas/5e4bd8f578963.pdf   
155  https://www.feiradesantana.ba.leg.br/admin/atas/5e4d3093e06f9.pdf   
165  https://www.feiradesantana.ba.leg.br/admin/atas/5e5d0167e9b52.pdf   
101  https://www.feiradesantana.ba.leg.br/admin/atas/5e5e525b29a58.pdf   
113  https://www.feiradesantana.ba.leg.br/admin/atas/5e5f9539996ef.pdf   
136  https://www.feiradesantana.ba.leg.br/admin/atas/5e68e1976cf52.pdf   
146  https://www.feiradesantana.ba.l

In [51]:
len(atas)

12

Doze atas mencionando termos que representam o COVID-19 foram encontradas.
Vamos ver o que elas dizem?

Atenção: o conteúdo das atas é grande para ser mostrado aqui. Por isso vamos
destacar as partes que contém os termos encontrados.

In [52]:
import re

padrao = r'[A-Z][^\\.;]*(coronavírus|covid)[^\\.;]*'


def trecho_encontrado(conteudo_do_arquivo):
    frases_encontradas = []
    for encontrado in re.finditer(padrao, conteudo_do_arquivo, re.IGNORECASE):
        frases_encontradas.append(encontrado.group().strip().replace('\n', ''))
    return '\n'.join(frases_encontradas)

atas['trecho'] = atas['file_content'].apply(trecho_encontrado)
pd.set_option('display.max_colwidth', 100)

atas[['date', 'event_type', 'title', 'file_urls', 'trecho']]

date        event_type                        title  \
100 2020-02-04  sessao_ordinaria   Ata da 2ª Sessão Ordinária   
135 2020-02-12  sessao_ordinaria   Ata da 5ª Sessão Ordinária   
145 2020-02-17  sessao_ordinaria   Ata da 6ª Sessão Ordinária   
155 2020-02-18  sessao_ordinaria   Ata da 7ª Sessão Ordinária   
165 2020-02-19  sessao_ordinaria   Ata da 8ª Sessão Ordinária   
101 2020-03-02  sessao_ordinaria   Ata da 9ª Sessão Ordinária   
113 2020-03-03  sessao_ordinaria  Ata da 10ª Sessão Ordinária   
136 2020-03-09  sessao_ordinaria  Ata da 12ª Sessão Ordinária   
146 2020-03-10  sessao_ordinaria  Ata da 13ª Sessão Ordinária   
156 2020-03-11  sessao_ordinaria  Ata da 14ª Sessão Ordinária   
166 2020-03-16  sessao_ordinaria  Ata da 15ª Sessão Ordinária   
176 2020-03-17  sessao_ordinaria  Ata da 16ª Sessão Ordinária   

                                                             file_urls  \
100  https://www.feiradesantana.ba.leg.br/admin/atas/5e46a0c72ab2c.pdf   
135  https://www.feiradesantana.ba.leg.br/admin/atas/5e4a7ec5b8961.pdf   
145  https://www.feiradesantana.ba.leg.br/admin/atas/5e4bd8f578963.pdf   
155  https://www.feiradesantana.ba.leg.br/admin/atas/5e4d3093e06f9.pdf   
165  https://www.feiradesantana.ba.leg.br/admin/atas/5e5d0167e9b52.pdf   
101  https://www.feiradesantana.ba.leg.br/admin/atas/5e5e525b29a58.pdf   
113  https://www.feiradesantana.ba.leg.br/admin/atas/5e5f9539996ef.pdf   
136  https://www.feiradesantana.ba.leg.br/admin/atas/5e68e1976cf52.pdf   
146  https://www.feiradesantana.ba.leg.br/admin/atas/5e70dd441eb06.pdf   
156  https://www.feiradesantana.ba.leg.br/admin/atas/5e70ddcf270a2.pdf   
166  https://www.feiradesantana.ba.leg.br/admin/atas/5e70de96a3dcc.pdf   
176  https://www.feiradesantana.ba.leg.br/admin/atas/5e7215c49ef90.pdf   

                                                                                                  trecho  
100                        Relatou, por fim, os cuidados com a higiene pessoal e mencionou o coronavírus  
135  Em seguida, o parlamentar Edvaldo Lima dos Santos (PP) comentou sobre solicitação feita por ele ...  
145  Posteriormente, o edil Edvaldo Lima dos Santos (PP) parabenizou todos os profissionais da impren...  
155  Logo após, o edil Edvaldo Lima dos Santos (PP) lembrou que agradeceu à imprensa feirense, por te...  
165  Dando continuidade, o edil Edvaldo Lima dos Santos (PP) registrou a resposta que recebeu do Mini...  
101  Com a palavra, o orador acentuou que solicitou, junto ao prefeito Colbert Martins da Silva Filho...  
113  Em seguida, o edil Isaías dos Santos (PDT) discorreu que os agentes de endemias e os agentes com...  
136  Na sequência, o edil Edvaldo Lima dos Santos (PP) relatou que, há cerca de 15 dias, ele solicito...  
146  Prosseguindo, o parlamentar José Carneiro Rocha (PSDB), em tempo cedido pela vereadora Aldney Ba...  
156  Em seguida, o orador à tribuna explicou que, em igrejas e escolas, o risco de contaminação pelo ...  
166  Amparado pelo Regimento Interno, o vereador José Carneiro Rocha, Presidente desta Casa, declarou...  
176  Disse que estas categorias permanecem       Feira de Santana CÂMARA  MUNICIPAL   Casa da Cidadan...

Já que não temos tantos dados assim (apenas 12 atas) podemos fazer parte dessa análise manual,
para ter certeza que nenhum vereador foi deixado de fora. Vamos usar o próximo comando
para exportar os dados para um novo CSV. Nesse CSV vai ter os dados filtrados por data
e também o trecho, além do conteúdo da ata completo.

In [53]:
def converte_para_arquivo(df, nome_do_arquivo):
    conteudo_do_csv = df.to_csv(index=False)  # convertemos o conteudo para CSV
    arquivo = open(nome_do_arquivo, 'w')  # criamos um arquivo
    arquivo.write(conteudo_do_csv)
    arquivo.close()

converte_para_arquivo(atas, 'analise-covid19-atas-camara.csv')

### Quem levantou a bola do COVID-19 na Câmara?

Uma planilha com a análise do quem-disse-o-que pode ser encontrada [aqui](https://docs.google.com/spreadsheets/d/1h7ioFnHH8sGSxglThTpQX8W_rK9cgI_QRB3u5aAcNMI/edit?usp=sharing).

## E o que dizem os diários oficiais?

No diário oficial do município você encontra informações sobre o que virou
realidade: as decretas, medidas, nomeações, vetos.

Em Feira de Santana os diários dos poderes executivo e legislativo estão juntos.
Vamos filtrar os diários do legislativo, separar por data, como fizemos com as
atas, e ver o que foi feito.

In [54]:
# esse arquivo pode ser baixado aqui: https://www.kaggle.com/anapaulagomes/dirios-oficiais-de-feira-de-santana
diarios = pd.read_csv('gazettes-28.03.2020.csv')
diarios = diarios[diarios['power'] == 'legislativo']

diarios["date"] = pd.to_datetime(diarios["date"])
diarios = diarios[diarios["date"].isin(pd.date_range("2020-01-01", "2020-03-28"))]
diarios = diarios.sort_values('date', ascending=True)  # aqui ordenados por data em ordem crescente

diarios.head()

crawled_at  \
105  2020-03-28 11:39:14.809207   
104  2020-03-28 11:39:14.816252   
103  2020-03-28 11:39:14.821502   
102  2020-03-28 11:39:14.827240   
101  2020-03-28 11:39:14.833804   

                                                                               crawled_from  \
105  http://www.diariooficial.feiradesantana.ba.gov.br/detalhes.asp?publicacao=2&edicao=700   
104  http://www.diariooficial.feiradesantana.ba.gov.br/detalhes.asp?publicacao=2&edicao=701   
103  http://www.diariooficial.feiradesantana.ba.gov.br/detalhes.asp?publicacao=2&edicao=702   
102  http://www.diariooficial.feiradesantana.ba.gov.br/detalhes.asp?publicacao=2&edicao=703   
101  http://www.diariooficial.feiradesantana.ba.gov.br/detalhes.asp?publicacao=2&edicao=704   

          date  \
105 2020-01-03   
104 2020-01-04   
103 2020-01-07   
102 2020-01-08   
101 2020-01-10   

                                                                                                  events  \
105  [{'title': 'EXTRATO DE DECRETOS', 'secretariat': 'CÂMARA MUNICIPAL', 'summary': 'Extrato dos Dec...   
104  [{'title': 'EXTRATO RESUMO ADITIVOS E CONTRATOS', 'secretariat': 'CÂMARA MUNICIPAL', 'summary': ...   
103        [{'title': 'PORTARIA', 'secretariat': 'CÂMARA MUNICIPAL', 'summary': 'Enquadrar servidores'}]   
102  [{'title': 'EXTRATO DE DECRETOS', 'secretariat': 'CÂMARA MUNICIPAL', 'summary': 'Extrato dos Dec...   
101  [{'title': 'DECRETO INDIVIDUAL', 'secretariat': 'CÂMARA MUNICIPAL', 'summary': 'PRORROGAÇÃO do A...   

                                                                                            file_content  \
105  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n      ...   
104  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n      ...   
103  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n...   
102  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n      ...   
101  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n      ...   

                                                                               file_urls  \
105  http://www.diariooficial.feiradesantana.ba.gov.br/atos/legislativo/2KWFRU212020.pdf   
104  http://www.diariooficial.feiradesantana.ba.gov.br/atos/legislativo/2HIKKR312020.pdf   
103  http://www.diariooficial.feiradesantana.ba.gov.br/atos/legislativo/2VSEYP612020.pdf   
102  http://www.diariooficial.feiradesantana.ba.gov.br/atos/legislativo/2SA2JH712020.pdf   
101  http://www.diariooficial.feiradesantana.ba.gov.br/atos/legislativo/2ESMIG912020.pdf   

           power        year_and_edition  
105  legislativo  Ano VI - Edição Nº 700  
104  legislativo  Ano VI - Edição Nº 701  
103  legislativo  Ano VI - Edição Nº 702  
102  legislativo  Ano VI - Edição Nº 703  
101  legislativo  Ano VI - Edição Nº 704

O que é importante saber sobre as colunas dessa base de dados:

* `date`: quando o diário foi publicado
* `power`: poder executivo ou legislativo (sim, os diários são unificados)
* `year_and_edition`: ano e edição do diário
* `events`:
* `crawled_at`: quando foi feita essa coleta
* `crawled_from`: qual a fonte dos dados
* `file_urls`: url dos arquivos
* `file_content`: o conteúdo do arquivo do diário

Vamos filtrar o conteúdo dos arquivos que contém os termos relacionados ao COVID-19
(os mesmos que utilizamos com as atas).

In [55]:
diarios = diarios[diarios['file_content'].str.contains(termos_covid, case=False)]
diarios['trecho'] = diarios['file_content'].apply(trecho_encontrado)

diarios[['date', 'power', 'year_and_edition', 'file_urls', 'trecho']]

date        power        year_and_edition  \
18 2020-03-17  legislativo  Ano VI - Edição Nº 733   
14 2020-03-25  legislativo  Ano VI - Edição Nº 737   
11 2020-03-26  legislativo  Ano VI - Edição Nº 738   
10 2020-03-27  legislativo  Ano VI - Edição Nº 739   

                                                                               file_urls  \
18  http://www.diariooficial.feiradesantana.ba.gov.br/atos/legislativo/2CI2L71632020.pdf   
14  http://www.diariooficial.feiradesantana.ba.gov.br/atos/legislativo/2S7S3K2432020.pdf   
11  http://www.diariooficial.feiradesantana.ba.gov.br/atos/legislativo/2JBCI22532020.pdf   
10  http://www.diariooficial.feiradesantana.ba.gov.br/atos/legislativo/2O9FOD2632020.pdf   

                                                                                                 trecho  
18  Estabelece medidas temporárias de prevenção ao contágio pelo Novo Coronavírus (COVID-19) conside...  
14                         Motivo: Medidas de prevenção contra a disseminação do coronavírus (COVID-19)  
11  A data para realização de uma nova sessão de abertura será publicada oportunamente, tendo em vis...  
10  A data para realização de uma nova sessão de abertura será publicada oportunamente, tendo em vis...

Apenas 4 diários com menção ao termo, entre 1 de Janeiro e 28 de Março de 2020
foram encontrados. O que será que eles dizem? Vamos exportar os resultados para
um novo CSV e continuar no Google Sheets.

In [56]:
converte_para_arquivo(diarios, 'analise-covid19-diarios-camara.csv')

### O que encontramos nos diários

Os 4 diários tratam de suspender licitações por conta da situação na cidade.
Apenas um dos diários, o diário do dia 17 de Março de 2020, [Ano VI - Edição Nº 733](http://www.diariooficial.feiradesantana.ba.gov.br/atos/legislativo/2CI2L71632020.pdf),
traz instruções a respeito do que será feito na casa. Aqui citamos o trecho que fala
a respeito das medidas:

> Art.  1º **Qualquer  servidor,  estagiário,  terceirizado,  vereador  que  apresentar  febre  ou  sintomas  respiratórios  (tosse  seca,  dor  de garganta,  mialgia,  cefaleia  e  prostração,  dificuldade  para  respirar  e  batimento  das  asas  nasais)  passa  a  ser  considerado  um  caso suspeito e deverá notificar imediatamente em até 24 horas à Vigilância Sanitária Epidemiológica/Secretaria Municipal de Saúde**.
§ 1ºQualquer servidor, estagiário, terceirizado, a partir dos 60 (sessenta)anos; portador de doença crônica e gestante estarão liberados das atividades laborais na Câmara Municipal de Feira de Santana sem prejuízo a sua remuneração.
§ 2º A(o) vereador(a) a partir dos 60 (sessenta)anos; portador de doença crônica e gestante será facultado as atividades laborais na Câmara Municipal de Feira de Santana sem prejuízo a sua remuneração
§ 3º Será  considerado  falta  justificada  ao  serviço  público  ou  à  atividade  laboral  privada  o  período  de  ausência  decorrente de afastamento por orientação médica.

> Art. 2º **Fica interditado o elevador no prédio anexo por tempo indeterminado**, até que sejam efetivamente contida a propagação do Coronavírus no Município e estabilizada a situação. _Parágrafo único_ O uso do elevador nesse período só será autorizado para transporte de portadores de necessidades especiais, como cadeirantes e pessoas com mobilidade reduzida.

> Art. 3º Será liberada a catraca para acesso ao prédio anexo. 

> Art. 4º O portãolateral (acesso a rua do prédio anexo) será fechado com entrada apenas pela portaria principal.

> Art. 5º Será disponibilizado nas áreas comuns dispensadores para álcool em gel.

> Art.6º Será  intensificada  a  limpeza  nos  banheiros,  elevadores,  corrimãos,  maçanetase  áreas  comuns  com  grande  circulação de pessoas.

> Art.7º Na cozinha e copa só será permitido simultaneamente à permanência de uma pessoa. 

> Art. 8º **Ficam suspensas as Sessões Solenes, Especiais e Audiências Públicas por tempo indeterminado**, até que sejam efetivamente contida a propagação do Coronavírus no Município e estabilizada a situação. 

> Art. 9º Nesse período de contenção é recomendado que o público/visitante assista a Sessão Ordinária on-line via TV Câmara. Parágrafo - Na  Portaria  do  Prédio  Sede  **haverá  um  servidor  orientando  as  pessoas  a  assistirem  os  trabalhos  legislativos  pela  TV Câmara** disponível  em https://www.feiradesantana.ba.leg.br/  distribuindo  panfletos  informativos  sobre  sintomas  e  métodos  de prevenção.

> Art. 10º No âmbito dos gabinetes dos respectivos Vereadores, **fica a critério de cada qual adotar restrições ao atendimento presencial do público externo ou visitação à sua respectiva área**.

> Art.11º A Gerência Administrativa fica autorizada a adotar outras providências administrativas necessárias para evitar a propagação interna do vírus COVID-19, devendo as medidas serem submetidas ao conhecimento da Presidência.

> Art.12º A validade desta Portaria será enquanto valer oestado de emergência de saúde pública realizado pela Secretaria Municipal de Saúde e pelo Comitê de Ações de Enfrentamento ao Coronavírus no Município de Feira de Santana.

> Art.13º Esta Portaria entra em vigor na data de sua publicação, revogadas disposições em contrário.


## Conclusão

Caso os edis da casa tenham feito mais nós não teríamos como saber sem ser por as atas,
pelo diário oficial ou por notícias do site. O ideal seria ter uma página com projetos
e iniciativas de cada vereador. Dessa forma, seria mais fácil se manter atualizado a
respeito do trabalho de cada um na cidade.

As análises manuais e o que encontramos pode ser visto [aqui](https://docs.google.com/spreadsheets/d/1h7ioFnHH8sGSxglThTpQX8W_rK9cgI_QRB3u5aAcNMI/edit?usp=sharing).
Um texto sobre a análise e as descobertas podem ser encontradas no blog do [Dados Abertos de Feira](https://medium.com/@dadosabertosdefeira).

Gostou do que viu? Achou interessante? Compartilhe com outras pessoas e não esqueça de mencionar
o projeto.